docs/user-guide/advanced/installation-options.md

```
@@ -0,0 +1,29 @@
+# Installation Options
+
+Install to a custom location:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)